### ISSUE TITLE
UI state body classes not ported

### ISSUE THREAD

**Issue by aertmann** [neos/neos-ui#1501]
Currently the main content area on the demo site is hidden behind the top bar so to add content to an empty page one has to use the content tree to select the collection.

![image](https://user-images.githubusercontent.com/903567/34393898-31d0d6ce-eb56-11e7-801c-bee352fecbe4.png)

This is due to the demo site making use of CSS adjustments for the backend to fix that, however the new UI doesn't have the same UI state classes added to the body tag as the old UI did.

Classes like `neos-backend` `neos-inspector-panel-open` `neos-navigate-panel-open` `neos-menu-panel-open` `neos-edit-preview-panel-open` `neos-full-screen` were used in the old UI to allow such modifications.

See https://github.com/neos/Neos.Demo/blob/master/Resources/Private/Styles/_Navigation.scss#L22-L44

**Comment by dimaip** [comment]
Yep I think this needs to be fixed. Just need to fix first-prio bugs and we'll get to this one.

**Comment by johannessteu** [comment]
I'd like to give this one a try!

**Comment by aertmann** [comment]
Nice @johannessteu. Just revisited this one and actually not sure if it even is needed at all since the requirements for the new UI are different due to the iFrame. Ideally one wouldn't need to alter anything on their website related to Neos UI as as long as it's responsive it should work.

The only thing might be related to the UI inside the iFrame (content element handles etc.), but that is already possible using the existing `neos-backend` class.

Looked into the demo site again and realized that the reason it's not working with the new UI is due to the alterations for the old UI, if those are removed it works as expected.

So would actually suggest to hold on with this one until the `Discussion` label I added has been resolved to see if it's even needed.
